libs/elasticsearch/pyproject.toml

```diff
@@ -58,6 +58,11 @@ optional = true
 [tool.poetry.group.test_integration.dependencies]
 
 
+
+[tool.poetry.group.codegen.dependencies]
+unasync = "^0.6.0"
+isort = "^5.13.2"
```

Comment:
I did not notice that the last time, but should we remove isort, as ruff includes it already?

Reply:
I'm just repeating what's in the Makefile:

https://github.com/langchain-ai/langchain-elastic/blob/686575ca13bd9b0936d38a3efed4803b47711f6e/libs/elasticsearch/Makefile#L30-L38